library/iojs

```diff
@@ -2,15 +2,24 @@
 
 1.0.4: git://github.com/iojs/docker-iojs@39ab80a90ff9addd935d3acf1f5bc3eff49d6ffc 1.0
```

Comment:
Is there a reason to keep `1.0`/`1.0.4` here?  For example, if a problem with `1.0.4` is discovered, will there be a `1.0.5`?

We won't delete the tags from the Hub unless it's explicitly requested by the maintainer that we do so, so people will still be able to use `iojs:1.0.4`, it just won't be listed under "Supported tags" in the description, and won't be rebuilt.